compiler_uv.sh

```diff
@@ -0,0 +1,128 @@
+#TODO add options for clean/noclean, make/nomake, cython/nocython
```

Comment:
I would suggest keeping a single "compiler" script and simply adding an option to switch between python installer utilities.